README.md

```diff
@@ -8,41 +8,49 @@ This project contains:
 - [lsp4jakarta](/lsp4jakarta) - Language Server for Jakarta EE
 - [jakarta-eclipse](/jakarta-eclipse) - Eclipse JDT LS extension and Eclipse IDE client that consumes the Language Server for Jakarta EE
 
+# Table of Contents  
+- [Getting Started](#Getting-Started)  
+- [Building](#Building)
+    - [Prerequisites](#Prerequisites)
+    - [Setup Instructions](#Setup-Instructions)
+    - [Common Errors](#Common-Errors)
```

Comment:
Remove these sections from the Table of Contents

Reply:
Opps, forgot about that!